tests/integration/hooks.go

```diff
@@ -40,8 +43,88 @@ var testObjectsTearDown = func(ctx context.Context, sc *godog.Scenario, _ error)
 	return ctx, nil
 }
 
-var istioCrTearDown = func(ctx context.Context, sc *godog.Scenario, _ error) (context.Context, error) {
+var additionalResourcesTearDown = func(ctx context.Context, sc *godog.Scenario, _ error) (context.Context, error) {
```

Comment:
I think we should add them here: https://github.com/kyma-project/istio/tree/main/manifests instead

Reply:
Or maybe putting an OwnerReference on the ConfigMaps would also work, so k8s will remove them for us

Reply:
yes, good idea, lets try that.